src/main/scala/org/apache/spark/sql/pulsar/PulsarConfigUpdater.scala

```diff
@@ -37,6 +38,10 @@ private[pulsar] case class PulsarConfigUpdater(
   def set(key: String, value: Object, map: ju.Map[String, Object]): this.type = {
     if (blacklistedKeys.contains(key)) {
       logInfo(s"$module: Skip $key")
+    } else if (keysToHideInLog.contains(key)) {
```

Comment:
Since the `keysToHideInLog` is just hide the log. How about drop this `else if` statement and merge them into one.

```scala
  def set(key: String,
          value: Object,
          map: ju.Map[String, Object]): this.type = {
    if (blacklistedKeys.contains(key)) {
      logInfo(s"$module: Skip $key")
    } else {
      map.put(key, value)
      logInfo(
        s"$module: Set $key to ${printConfigValue(key, Option(value))}," +
          s" earlier value: ${printConfigValue(key, pulsarParams.get(key))}"
      )
    }
    this
  }


  private val HIDE_COMPLETELY_LIMIT = 6
  private val SHOW_FRACTION_OF_HIDDEN_VALUE = 1.0 / 3.0

  private def printConfigValue(key: String,
                               maybeVal: Option[Object]): String = {
    val value = maybeVal.map(_.toString).getOrElse("")
    if (keysToHideInLog.contains(key)) {
      if (value.length < HIDE_COMPLETELY_LIMIT) {
        return "...<completely hidden>..."
      } else {
        return s"${value.take((value.length * SHOW_FRACTION_OF_HIDDEN_VALUE).toInt)}..."
      }
    }

    value
  }
```

Reply:
I have changed the code accordingly.